.github/workflows/unit-tests.yml

```diff
@@ -1,17 +1,11 @@
 name: PHP Unit Tests
 
 on:
-  push:
-    branches:
-      - feature/6.x/unit-tests
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
     branches:
-      - 6.dev
-      - 6.2.0
-      - 6.3.0
-      - 6.4.0
-      - 6.5.0
+      - '*.dev'
```

Comment:
@intoeetive Is this okay, or should it be '6.dev'. I'm not really sure if it makes a difference.